sympy/physics/control/control_plots.py

```diff
@@ -736,18 +737,25 @@ def bode_magnitude_numerical_data(system, initial_exp=-5, final_exp=5, **kwargs)
 
 
 def bode_magnitude_plot(system, initial_exp=-5, final_exp=5,
-    color='b', show_axes=False, grid=True, show=True, **kwargs):
+    color='b', show_axes=False, grid=True, show=True, Freq_unit='rad/sec', **kwargs):
```

Comment:
Use lower case `freq_unit` (same for keyword arguments of the other function below).